### PR TITLE
Tests for Game Mechanics

### PR DESCRIPTION
**Pull request by cheatiiit**

### CODE REVIEW

**Comment by cheatiiit** [comment]
@alma-castant : Could you please help me. I am not able to understand how to initialize the objects of the Activity by invoking a mock intent.

**Comment by alma-castant** [comment]
You have created this branch from a previous non-merged branch and you have taken all its commits along. 
Fetch the changes from the develop branch in this repo into your repo so it's up to date and create a new branch from there to include these changes. Create one commit only and amend it using `git commit --amend` if there's something you need to fix

File: Android/PowerUpApp/PowerUpApp/src/com/example/powerupapp/datamodel/Score.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.example.powerupapp.datamodel;
+
+public class Score {
+	public Integer healing;
+	public Integer strength;
+	public Integer telepathy;
+	public Integer invisibilty;
```
Comment (alma-castant) [inline Android/PowerUpApp/PowerUpApp/src/com/example/powerupapp/datamodel/Score.java]: There is a typo in invisibility here, and in its getter and setter

**Comment by alma-castant** [comment]
Make sure you include the new logo with bigger resolution in the main screen (let me know if the resolution is not enough yet)

**Comment by alma-castant** [comment]
Place the "Power Up" text in the main screen lower. It currently shows on the frame on a Nexus 5 emulator

**Comment by alma-castant** [comment]
Make sure the text in the quizzes doesn't shown on top of the frame

**Comment by alma-castant** [comment]
When I press "Start" on the main screen I go to a blank page, and when I press "New user" I go to the map

**Comment by cheatiiit** [comment]
This PR is not worth changing. The code is out of date and many changes have been made in the other PR

**Comment by rosariorobinson** [comment]
??? What is next steps?

**Comment by cheatiiit** [comment]
Need to close this PR. This should not be merged. Thanks :)

**Pull request closed without merging**